### PR TITLE
Copy tools to spatial dir

### PR DESCRIPTION
**Pull request by yumnuska**
#### Description
Copy schema compiler and snapshot converter to spatial/tools directory for easy access.

### CODE REVIEW

**Comment by yumnuska** [comment]
Feel free to merge this if it looks good.

File: Setup.bat [hunk 1]
```diff
@@ -60,9 +60,10 @@ call :MarkStartOfBlock "Setup variables"
     set BINARIES_DIR=%~dp0SpatialGDK\Binaries\ThirdParty\Improbable
 
     rem Copy schema to the projects spatial directory.
-    set SCHEMA_COPY_DIR=%~dp0..\..\..\spatial\schema\unreal\gdk
-    set SCHEMA_STD_COPY_DIR=%~dp0..\..\..\spatial\build\dependencies\schema\standard_library
     set SPATIAL_DIR=%~dp0..\..\..\spatial
+    set SPATIAL_TOOLS_COPY_DIR=%SPATIAL_DIR%\tools
```
Comment (m-samiec) [inline Setup.bat:64]: Is this necessary over just using them in their default extracted location?

Comment (yumnuska) [reply Setup.bat:64]: strictly, no, although it's much more convenient.

Comment (yumnuska) [reply Setup.bat:64]: compare `spatial\tools\snapshot_converter` to `Game\Plugins\UnrealGDK\SpatialGDK\Binaries\ThirdParty\Improbable\Programs\snapshot_converter`

Comment (m-samiec) [reply Setup.bat:64]: Yep fair enough!

**Review by m-samiec — APPROVED** [review]

**Pull request merged**